packages/core/src/SimpleTable/index.tsx

```diff
@@ -80,12 +92,16 @@ const SimpleTableHeader: React.FC<SimpleTableHeaderProps> = ({
   hasScrollSpace = false,
   children,
 }) => {
-  const { compact } = useContext(SimpleTableContext)
+  const { compact, widths } = useContext(SimpleTableContext)
 
   return (
     <TableHeader compact={compact}>
       {React.Children.map(children, (cell, i) => {
-        return <TableHeaderCellContent key={i}>{cell}</TableHeaderCellContent>
+        return (
+          <TableHeaderCellContent key={i} width={widths[i]}>
```

Comment:
An array doesn't throw an error if you try to access an index that doesn't exist? Ie, if you didn't specify a widths array.

Reply:
If you didn't specify a widths array, then it will be an empty array. And I think the array just returns undefined if it try to access an index that doesn't exist. Is it wrong?
![Screenshot from 2021-11-17 13-39-53](https://user-images.githubusercontent.com/24866179/142202746-aa6a780d-0ae5-4c65-bf4f-cb72de450f88.png)

Reply:
No that's fine. I just couldn't double check since I'm not at my computer.